src/routes/meteors.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { getMeteors, getStartPage } from '../controllers/meteors';
+import validateRequestSchema from '../apiSchemas/request/schemaValidator';
+
+const router = express.Router();
+
+router.get('/', getStartPage);
+router.get('/meteors', validateRequestSchema('neoWsSchema'), getMeteors);
```

Comment:
As I mentioned in the comment above, we can pass schema object instead of string and avoid extra if. 
`router.get('/meteors', validateRequestSchema(neoWsSchema), getMeteors);`